test-core/src/main/java/io/micronaut/test/annotation/MicronautTest.java

```diff
@@ -91,4 +91,14 @@
      * @return The builder
      */
     Class<? extends ApplicationContextBuilder>[] contextBuilder() default {};
+
+    /**
+     * <p>Whether to start {@link io.micronaut.runtime.EmbeddedApplication}.</p>
+     *
+     * <p>When false, only the application context will be started.
+     * This can be used to disable {@link io.micronaut.runtime.server.EmbeddedServer}.</p>
+     *
+     * @return true if {@link io.micronaut.runtime.EmbeddedApplication} should be started
+     */
+    boolean embeddedApplication() default true;
```

Comment:
Can you rename this to `startApplication()`, I think the name `EmbeddedApplication` is more an implementation detail and `startApplication` would have a clearer purpose